lib/ui/tiles/bookmark_button.dart

```diff
@@ -12,7 +13,8 @@ class BookmarkButton extends StatefulWidget {
 
 class _BookmarkButtonState extends State<BookmarkButton> {
   bool marked;
-
+  final _snackBar =
+      SnackBar(content: Text('Web does not support this feature'));
```

Comment:
Update message: Bookmarking is not supported on web.
Also add a "Bookmark" icon along with the meessage.

Reply:
Done